lib/web-service/web-service.ts

```diff
@@ -421,15 +425,14 @@ export class WebService extends Construct {
         "Either cpuTargetUtilization or memoryTargetUtilization must be specified to use a horizontalPodAutoscaler",
       );
     }
-
-    if (props.podDisruptionBudget) {
-      this.validatePodDisruptionBudget(props);
-    }
   }
 
-  private validatePodDisruptionBudget(props: WebServiceProps): void {
+  private validatePodDisruptionBudget(
+    podDisruptionBudget: PodDisruptionBudgetProps,
+    props: WebServiceProps,
+  ): void {
     const { minAvailable, maxUnavailable } =
-      props.podDisruptionBudget as PodDisruptionBudgetProps;
+      podDisruptionBudget as PodDisruptionBudgetProps;
```

Comment:
No need to force the type because it's already set in the function argument
```suggestion
      podDisruptionBudget;
```